pkg/mcs/scheduling/server/cluster.go

```diff
@@ -81,6 +91,9 @@ func NewCluster(parentCtx context.Context, persistConfig *config.PersistConfig,
 		storage:           storage,
 		clusterID:         clusterID,
 		checkMembershipCh: checkMembershipCh,
+
+		taskRunner:           ratelimit.NewConcurrentRunner(hbConcurrentRunner, time.Minute),
```

Comment:
Will it cause the memory increase if there are many hb requests?

Reply:
it depends on the number of waiting queue closures (tasks). we suppose each costs 128 bytes, 1M is about 128MB. but it's released very quickly I think.